.circleci/config.yml

```diff
@@ -114,8 +114,7 @@ commands:
           at: .
```

Comment:
There are a lot of necessary formatting changes here that we should revert (might still be good to do them, just in a separate PR).

Reply:
I don't think I meant to do them, I don't really know how it happened? I only added aioredis stuff and committed the changes. I don't know what might've changed the formatting. @Kyle-Verhoog